sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -277,6 +277,11 @@ struct _pi_queue : _pi_object {
       : ZeCommandQueue{Queue}, Context{Context}, Device{Device},
         QueueBatchSize{QueueBatchSize} {}
 
+  typedef enum {
```

Comment:
Please add comments about the type and the values.

Reply:
I got rid of the need for this enum, and deleted it.